client/src/components/Map.tsx

```diff
@@ -113,20 +128,19 @@ export default class Map extends Component<Prop>{
         
         d3.selectAll("circle").remove().exit()
         d3.selectAll("line").remove().exit()
+        d3.selectAll("marker").remove().exit()
         this.connectNodes()
     }
 
     render() {
-    
+
         return(
-                <div className="w-full h-2/5 border max-w-3xl mx-auto">
-                    <ZoomableSVG width={this.props.width} height={this.props.height} 
-                        centroidCrop={this.props.centroidCrop} enableZoom={false}>
-                        <svg ref={(mapRef: SVGSVGElement) => this.mapRef = mapRef}>
-                            
-                        </svg>
-                    </ZoomableSVG>
-                </div>
+            <ZoomableSVG width={this.props.width} height={this.props.height} 
+                centroidCrop={this.props.centroidCrop} enableZoom={this.props.enableZoom}>
+                <svg ref={(mapRef: SVGSVGElement) => this.mapRef = mapRef}>
+                    
+                </svg>
+            </ZoomableSVG>
```

Comment:
Great work! I wanted to suggest this change before. The Map component shouldn't force its maximum size, but rather the user of the Map (navigation page in this example). This also improved the /graph page because each submap is full width.